Core/Shared/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_subdirectory(JsRuntime)
+add_subdirectory(AppRuntime)
+add_subdirectory(ScriptLoader)
```

Comment:
Is this file supposed to be here? This feels like it might have been left over from something.

Reply:
Oops. Yeah, this is not supposed to be here.